pkg/verifier/aws/aws_verifier.go

```diff
@@ -325,3 +327,120 @@ func (a *AwsVerifier) writeDebugLogs(log string) {
 	a.Output.AddDebugLogs(log)
 	a.Logger.Debug(context.TODO(), log)
 }
+
+// CreateSecurityGroup creates a security group with the specified name and cluster tag key in a specified VPC
+func (a *AwsVerifier) CreateSecurityGroup(ctx context.Context, name, vpcId string) (*ec2.CreateSecurityGroupOutput, error) {
+	input := &ec2.CreateSecurityGroupInput{
+		GroupName:   awsTools.String(name),
+		VpcId:       &vpcId,
+		Description: awsTools.String("osd-network-verifier security group"),
```

Comment:
Please add `TagSpecification` and fill with the tags provided

Reply:
Part of the latest commit